stellar-multitenant/pkg/cli/add_tenants.go

```diff
@@ -23,18 +28,20 @@ type AddTenantsCommandOptions struct {
 	SDPUIBaseURL     *string
 	NetworkType      string
 	MessengerOptions message.MessengerOptions
+	HorizonURL       string
 }
 
 func AddTenantsCmd() *cobra.Command {
-	opts := AddTenantsCommandOptions{}
+	tenantsOpts := AddTenantsCommandOptions{}
+	sigOpts := signing.SignatureServiceOptions{}
 
-	configOptions := config.ConfigOptions{
+	configOptions := append(config.ConfigOptions{
```

Comment:
In other cmd files, we are adding the configOptions slices right after we declare the CommandOptions, like in:

* [cmd/serve.go](https://github.com/stellar/stellar-disbursement-platform-backend/blob/c1d1af01125f2a26efb16c4abef5fd785bdb0141/cmd/serve.go#L310-L357)
* [cmd/transaction_submitter.go](https://github.com/stellar/stellar-disbursement-platform-backend/blob/c1d1af01125f2a26efb16c4abef5fd785bdb0141/cmd/transaction_submitter.go#L67-L122)

Could you plz follow this project pattern here as well?